frontend/src/components/ViewTrips/save-trip-form-elements.js

```diff
@@ -91,9 +91,9 @@ function createEmailFormControl(ref, idx, placeholder, defaultEmailArr) {
  * @param {boolean} isAddTripForm True if form is adding new trip, false if
  *     form is editting existing trip.
  * @param {string} placeholder Placeholder text value in the form input.
- * @param {?Array<string>} defaultEmailArr Array of the emails to be displayed
+ * @param {?string[]} defaultEmailArr Array of the emails to be displayed
  *     in the default form fields.
- * @return {JSX.Element} The Form.Control elements.
+ * @return {Array<JSX.Element>} The array of Form.Control elements.
```

Comment:
```suggestion
 * @return {JSX.Element[]} The array of Form.Control elements.
```
?

Reply:
I can change this if you think it looks better. But my though process was use the `type[]` syntax whenever `type` was one word like string or number. But in instances like `JSX.Element` I think the other syntax looks a little bit better.

Reply:
Does the JSDoc documentation say whether there is a preference? Or is it all just up to the person implementing?